src/main/java/Duke.java

```diff
@@ -1,10 +1,282 @@
+import duke.task.Deadline;
+import duke.exceptions.DukeException;
+import duke.exceptions.EmptyInputException;
+import duke.exceptions.IllegalInputException;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
 public class Duke {
+    public static final String LINE = "____________________________________________________________\n";
+    public static final String BYE_MESSAGE = "Bye. Hope to see you again soon!\n";
+    public static boolean canExit = false;
+
+    /**
+     * Prints message after adding To-Do/Deadline/Event duke.task
+     *
+     * @param myList
+     */
+    public static void printTaskMessage(ArrayList<Task> myList) {
+        System.out.println(LINE
+                + "Got it. I've added this duke.task:\n"
+                + myList.get(myList.size() - 1) + "\n"
+                + "Now you have " + myList.size() + " in the list.\n"
+                + LINE);
+    }
+
+    /**
+     * Prints a greeting user message
+     */
+    public static void printGreetMessage() {
+        System.out.println(LINE + "Hello! I'm Duke\n"
+                + "What can I do for you?\n"
+                + LINE);
+    }
+
+    /**
+     * Prints exiting message before program stops
+     */
+    public static void printByeMessage() {
+        System.out.println(BYE_MESSAGE);
+    }
+
+    /**
+     * Prints an added message
+     *
+     * @param myList
+     */
+    public static void printAddedTaskMessage(ArrayList<Task> myList) {
+        System.out.println(LINE + "added: "
+                + myList.get(myList.size() - 1).getDescription()
+                + "\n" + LINE);
+    }
+    public static void printIllegalInputMessage() {
+        System.out.println(LINE
+                +  "☹ OOPS!!! I'm sorry, but I don't know what that means :-(\n"
+                + LINE);
+    }
+    public static void printEmptyInputMessage(String task) {
+        System.out.println(LINE
+                + "☹ OOPS!!! The description of " + task + " cannot be empty.\n"
+                + LINE);
+    }
+    /**
+     * Adds the input text into the list
+     *
+     * @param s
+     * @param myList
+     */
+    public static void addList(String s, ArrayList<Task> myList) {
+        Task t = new Task(s);
+        myList.add(t);
+    }
+
+    /**
+     * Prints all the items in the list
+     *
+     * @param myList
+     */
+    public static void printList(ArrayList<Task> myList) {
+        System.out.println(LINE + "Here are the tasks in your list: \n");
+        for (int i = 0; i < myList.size(); ++i) {
+            System.out.println(i + 1 + ". " + myList.get(i));
+        }
+        System.out.println(LINE);
+    }
+
+    /**
+     * Marks the given item in the list with an "X"
+     *
+     * @param s
+     * @param myList
+     */
+    public static void markDone(String s, ArrayList<Task> myList) {
+        String taskToMarkString = s.substring(s.length() - 1);
+        int taskToMark = Integer.parseInt(taskToMarkString) - 1;
```

Comment:
Since you are referring to the index, maybe your variable name can be `taskNumberToMark`?